src/imageio/format/png.c

```diff
@@ -52,7 +52,7 @@ typedef struct dt_imageio_png_gui_t
   GtkWidget *compression;
 } dt_imageio_png_gui_t;
 
-/* Write EXIF data to PNG file.
+/* Write EXIF data to PNG file (legacy, before PNG extension for eXIf chunk).
```

Comment:
[Exiftool actually uses this one now as well](https://exiftool.org/TagNames/PNG.html):

"this is where ExifTool will create new EXIF"

Reply:
Removed this commit as it lead to eXIf being written twice (and Xmp writing by exiv2 failing), so will follow up separately.